src/core/manager.c

```diff
@@ -870,7 +870,7 @@ int manager_new(UnitFileScope scope, ManagerTestRunFlags test_run_flags, Manager
         if (r < 0)
                 return r;
 
-        if (test_run_flags == MANAGER_TEST_RUN_MINIMAL) {
+        if (FLAGS_SET(test_run_flags, MANAGER_TEST_RUN_MINIMAL)) {
```

Comment:
there are some other places where we check for the flag with ==. Shouldn't they be updated too?

Reply:
Yes there's two more, but they seemed more intentional, judging by which tests are setting the flag to be exactly == MANAGER_TEST_RUN_MINIMAL, so I opted to leave them as-is as I was not 100% sure